src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -58,7 +91,7 @@ public class CoinstakeWorkerContext
             public ILogger Logger { get; set; }
 
             /// <summary>List of UTXO descriptions that the worker should check.</summary>
-            public List<StakeTx> Coins { get; set; }
+            public List<StakeTx> Transactions { get; set; }
```

Comment:
as explained on slack, this change is not correct imho
Coin != Money, Coin is UTXO
i would stay with the terminology that is being used, so i would not make any changes here, but if i were not to use Coin, i'd go with something like "UtxoStakeInfo" or something like that, but then Coin is better again :-)
but also note that StakeTx is not a good name for that class

Reply:
Why not? StakeTx immediately convey the purpose of the class. But I am very happy with UtxoStakeInfo as well.

Reply:
StakeTx is wrong name for the class and also Transactions is wrong name of this list
Coins or StakingCoins is probably the best for this list and we need to decide how to rename StakeTx because StakeTx is no transaction, it even contains wallet password ...

Reply:
Hmm yeah, StakingCoins sounds right to me it does indeed represent more then just a UTXO.
So class should be `StakingCoin`  and the list `StakingCoins`

Reply:
ok, cool, now the thing is that @noescape00 went forward and renamed StakeTx to UtxoStakeDescription and also modified some comments (but not all of them!! :D ) as such

so the question is, are we happy with UtxoStakeDescription instead of StakingCoin? or do we want to tease @noescape00 little more and let him redo?

Reply:
i mean everyone has to have a limit when they break ;) can we find one for @noescape00 :D

Reply:
@noescape00 I will buy you a beer when you come to London if you survive @Aprogiena code reviews.

Reply:
`UtxoStakeDescription` is not a bad name either, so I will leave it to @noescape00 to decide.

Reply:
ok cool, if you don't mind, i don't mind, so i will also leave it to @noescape00

Reply:
>i mean everyone has to have a limit when they break ;) can we find one for @noescape00 :D

You can try =) 

>@noescape00 I will buy you a beer when you come to London if you survive @Aprogiena code reviews.

That's a deal! 

>now the thing is that @noescape00 went forward and renamed StakeTx to UtxoStakeDescription

The reason for that is that we have a lot of lists so naming them utxoStakeInfos would have looked weird.

Reply:
that's fine, let's leave it as it is UtxoStakeDescription

but the point was missed, so i just explain that (but we won't be doing any changes) - what we were discussing is that we don't really use UTXO in variables/classes names, we use Coin. When you see Coin, you know it is UTXO, but it is just everywhere else called Coin (CoinView for example). So we were suggesting having StakingCoin class instead of StakeTx class. We have UtxoStakeDescription now, which is fine, just very slightly inconsistent with the rest of the code base, but still very fine because the coin IS UTXO, so there is no harm. 

so this is just for future - if you see "coin" think about UTXO, that's all. so when you see CoinView, you think some kind of view of UTXOs, and in fact it is some perspective of the UTXO set

Reply:
>so this is just for future - if you see "coin" think about UTXO, that's all. so when you see CoinView, you think some kind of view of UTXOs, and in fact it is some perspective of the UTXO set.

Yep, got it. 

Though not sure why btc core guys came with this name in the first place, that was really confusing.

Reply:
It seems so, but then it is not :-) 
If you think about it, these cryptocurrencies have coins, so what exactly is the coin? The coin is something you can prove ownership to and you can send to someone else. In Bitcoin, that's basically UTXO. So after all, UTXO is the coin.

Reply:
That's a good way to put it, actually. Now I see the logic behind it.